src/components/FrontPage/TeacherInfobox.scss

```diff
@@ -6,24 +6,15 @@
   border-bottom: 1px black dotted;
 }
 
-.fontInfoBox {
-  font-family: $font-family-roboto;
-  font-weight: 300;
-}
-
-.infoBoxContainer {
-  display: flex;
-  justify-content: center;
-}
-
 .infoBox {
-  composes: fontInfoBox;
-  padding: 5px;
-  margin: 0 16px;
+  padding: 10px 0;
   max-width: 700px;
   background-color: #f7f7f7 !important;
   border: 1px solid #b2b2b2;
   border-radius: 10px;
+  font-family: $font-family-roboto;
+  font-size: 12pt;
```

Comment:
Kanskje spesifisere dette som `em` i stedet for `pt`? Se kommentar i _general.scss. Om du ikke har vært gjennom diskusjonen om hvilken enhet man bruker på font, så kan følgende også være god lesning:
http://kyleschaeffer.com/development/css-font-size-em-vs-px-vs-pt-vs/

Vær oppmerksom på at dette er en litt gammel artikkel, så det er sikkert lurt å google for litt nyere diskusjoner også.

Men kort fortalt: Dersom det er viktig at teksten er akkurat den størrelsen, selv om man skulle endre base-font-størrelsen, så bruk `px`. Dersom man ønsker at denne teksten skal endre seg dersom man endrer base-font-størrelsen, så bruk `em`. Det siste er som regel det man vil. Dersom man trenger at font-størrelsen endrer seg som følge av forskjellige browser-størrelser, bruk media-queries. Uansett, unngå `pt`.